sources/account_abstraction/erc4337/b3/account_abstraction_erc4337_b3_sources.yml

```diff
@@ -0,0 +1,30 @@
+version: 2
+
+sources:
+  - name: erc4337_b3
+    description: >
+      Decoded tables related to erc4337.
+    tables:
+      - name: EntryPoint_v0_6_evt_UserOperationEvent
+        description: >
+          Decoded table related to the erc4337 user operation events version v0.6.
+
+      - name: EntryPoint_v0_6_call_handleOps
+        description: >
+          Decoded table related to the erc4337 handle operation calls, version v0.6.
+
+      - name: EntryPoint_v0_6_evt_AccountDeployed
+        description: >
+          Decoded table related to the erc4337 account deployed events, version v0.6.
```

Comment:
We only have v0.7 online at the moment.
Seems v0.6 is missing yet.

Reply:
decoding pipeline screwing me over once again 🙃